src/editors/Flow/view/Components/FlowTopBar/FlowTopBar.jsx

```diff
@@ -40,7 +40,6 @@ import { buttonStyles, flowTopBarStyles } from "./styles";
 import FlowSearch from "./FlowSearch";
 
 const BACKEND_CALLBACK_NAME = "backend.FlowTopBar";
-const FEEDBACK_TIMEOUT = 10000;
```

Comment:
Did you investigate why was this needed in the first place?
Are we sure that removing this won't cause any more regressions?

Reply:
It should have never been there. The FE shouldn't be putting timeouts like those.

The timeout seems to have been put there because the person didn't think that it could take more than 10 seconds. Among other things.

Reply:
Do you know the feeling, when you do some work and after a few hours of work you know it's done? You never think about the other people that did work prior to you, you are concentrated on your task and you just go in and try to fix it. In your opinion, people that did code before you, just did it bad. 
There's a reason someone wasted their time to add that FEEDBACK_TIMEOUT. It wasn't created out of thin air, it wasn't some idiot that hacked mov.ai and put that there out of spite or something. It was done by a movai developer that had a task, just like you, and tried to solve it. 

I'm just asking if you investigated why this was needed, why that person a couple of years ago thought about adding that there, why 2 more people looked at the code and accepted that. 

It would be better, if they left a comment explaining why the hell is that needed, but they didn't, so it's up to us to investigate if it's still needed or not. If not, you have my blessing, remove it please! I hate these timeouts. BUT, if it's there, there was a reason to add it. Just do your part, do the investigation and let us know why that was added and why it is no longer needed.

Reply:
So perhaps I'm the only person who can make mistakes, then.
Is it possible for you to be missing something?

Reply:
> It should have never been there. The FE shouldn't be putting timeouts like those.
> 
> The timeout seems to have been put there because the person didn't think that it could take more than 10 seconds. Among other things. Like the fact that kind of thing shouldn't be done.

Is there any resource or tests done that this won't cause any type of regressions?

Reply:
Stop talking about yourself. I just want to find out why it was added in the first place. Manuel built this application, or most of it, and he added that timeout there for a reason, perhaps it was because the backend was slower or the application didn't work fine at that time. I just asked if it was safe to remove that. Nothing else. I don't care if I have potencial or not, or whatever you're rambling about. I asked a question, you're supposed to dig in and understand if there might be something breaking if that Timeout is removed or not, and tell me: "It really isn't needed anymore because of X reason", or "Good thing you asked, after digging a bit I found out why it was added in the first place and we should still use it because of X or Y reason."
This is all I'm asking.

Reply:
Ok, Then.

It was added because the developer then seemed to think it was sufficient to wait for a fixed timeout on the FE side to find out whether or not the flow had started or stopped.

It's not.

Reply:
I don't believe so, Nadia. But it does work.